README.md

```diff
@@ -0,0 +1,80 @@
+Signing HTTP Messages
+=====================
+
+This project is the main work area for the [Signing HTTP Messages draft
+standard](https://datatracker.ietf.org/doc/draft-cavage-http-signatures/).
+
+The standard aims to provide a way for senders of HTTP messages - both
+requests and respnses - to digitally sign specified headers in a manner
+compatible with existing HTTP implementations.
+
+# Motivation
+
+Many schemes exist to protect data objects that may be transported as the
+payload of a HTTP message. These include JWS, XML Signatures, and detached
+sgnatures in multipart messages using e.g. GPG.
+
+However there is no accepted mechanism to provide protection to the HTTP
+message as a whole, and especially within a single HTTP request/response
```

Comment:
imho we are protecting part of the http message. 

we are not protecting the http message as a whole. If once we send a message, an intermediary
decides to apply transfer-coding and deliver that via multiple messages, he can do that.

Moreover we don't have a way to express that some headers MUST NOT be present, so the message is changed.

Reply:
That's why I say "mechanism". This protocol cannot protect an entire message, but in conjunction with payload digests it can, which we should illustrate but not consider core.

Reply:
I'll try to PR next week: hope to clarify.